ADOPTERS.md

```diff
@@ -35,3 +35,4 @@ This list is sorted in the order that organizations were added to it.
 | [Renault](https://www.renaultgroup.com/) | @smileisak | Building Renault Kubernetes Platform resources using XRDs and compositions for an additional layer of abstraction to provide end-user services. |
 | [Gympass](https://gympass.com) | @caiofralmeida | Builds a self-service platform so engineers can be more productive in resource provisioning. |
 | [Deutsche Kreditbank AG](https://www.dkb.de/) | @haarchri | Developer Self-Service for DKB Standard Operating Platform, higher standardization, simplification of tech stack and continuous reconciliation|
+| [Wildlife Studios](https://wildlifestudios.com/) | @kasama | Component that powers an API used to manage infrastructure for a self-service platform for developers |
```

Comment:
Is this platform being used in a production environment? If so, feel free to explicitly add that note here too 😁